docs/README.md

```diff
@@ -0,0 +1,186 @@
+# quarkus-restlight
+
+`Quarkus-restlight` is an extension of `esa-restlight`. It can use the `quarkus` framework to help users compile `restlight` applications into `native image`, shorten the startup time to milliseconds, and greatly reduce the memory usage, while reducing the image size from 300M+ to 10M.
```

Comment:
The `300MB+` number has no precise numerical backing and should probably be avoided.